kymatio/scattering1d/frontend/base_frontend.py

```diff
@@ -62,11 +61,11 @@ def build(self):
         self.log2_T = math.floor(math.log2(self.T))
 
         # Compute the minimum support to pad (ideally)
-        min_to_pad = compute_minimum_support_to_pad(
-            self.N, self.J, self.Q, self.T, r_psi=self.r_psi, sigma0=self.sigma0,
-            alpha=self.alpha, P_max=self.P_max, eps=self.eps,
-            criterion_amplitude=self.criterion_amplitude,
-            normalize=self.normalize)
+        phi_f = gauss_1d(
+            self.N, self.sigma0/self.T, self.normalize, self.P_max, self.eps)
+        min_to_pad = 3 * compute_temporal_support(
+            phi_f.reshape(1, -1), criterion_amplitude=1e-3)
+
```

Comment:
perhaps these two lines should be combined into `compute_temporal_support`

Reply:
that's the next step, yes ;)